### PR TITLE
fix: discrepancies in 0.13.0 fee docs

### PR DESCRIPTION
**Pull request by ArielElp**
### Description of the Changes

Fixing a few discrepancies in 0.13.0 fee docs

### PR Preview URL

After you push a commit to this PR, a preview is built and a URL to the root of the preview appears in the comment feed.

https://starknet-io.github.io/starknet-docs/pr-1078/documentation/architecture_and_concepts/Network_Architecture/fee-mechanism/

### Check List

- [ ] Changes have been done against main branch, and PR does not conflict
- [ ] PR title follows the convention: `<docs/feat/fix/chore>(optional scope): <description>`, e.g: `fix: minor typos in code`

- - -
This change is [Reviewable](https://reviewable.io/reviews/starknet-io/starknet-docs/1078)

### CODE REVIEW

**Review by iliav-starkware — COMMENTED** [review]
> Reviewed 1 of 1 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/starknet-io/starknet-docs/1078)* status: <span>:shipit:</span> complete! all files reviewed, all discussions resolved (waiting on @stoobie)

**Review by iliav-starkware — COMMENTED** [review]
:lgtm:

> *[Reviewable](https://reviewable.io/reviews/starknet-io/starknet-docs/1078#-:-No6itlvE7O4JrhXaDZS:bnfp4nl)* status: <span>:shipit:</span> complete! all files reviewed, all discussions resolved (waiting on @stoobie)

File: components/Starknet/modules/architecture_and_concepts/pages/Network_Architecture/fee-mechanism.adoc [hunk 1]
```diff
@@ -38,7 +38,8 @@ For more information see xref:#calculation_of_computation_costs[Calculation of c
 * stem:[$\ell$] is the number of contract deployments.
 * stem:[$0.9$] is a 10% cost reduction that considers that the sequencer does not incur additional costs for repeated updates to the same storage slot within a single block.
 * stem:[$w$] is the xref:#calculation_of_computation_costs[`CairoResourceFeeWeights`] vector.
-* stem:[$\text{calldata_cost}$] is 512 gas per 32-byte word.
+* stem:[$\text{calldata_cost}$] is 612 gas per 32-byte word (~100 gas is charged for on-chain hashing the happens for every sent word).
+* stem:[$\text{storage_write_cost}$] is the 20,000 gas, the cost associated with allocating a new storage cell on Ethereum.
```
Comment (stoobie) [inline components/Starknet/modules/architecture_and_concepts/pages/Network_Architecture/fee-mechanism.adoc:42]:
```suggestion
* stem:[$\text{storage_write_cost}$] is 20,000 gas, the cost of allocating a new storage cell on Ethereum.
```

**Review by stoobie — CHANGES_REQUESTED** [review]
One minor grammatical change requested.

**Review by iliav-starkware — COMMENTED** [review]
> Reviewed 1 of 1 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/starknet-io/starknet-docs/1078)* status: all files reviewed, 1 unresolved discussion (waiting on @ArielElp)

**Review by iliav-starkware — COMMENTED** [review]
> Reviewed all commit messages.
> *[Reviewable](https://reviewable.io/reviews/starknet-io/starknet-docs/1078)* status: all files reviewed, 1 unresolved discussion (waiting on @ArielElp)

**Review by stoobie — APPROVED** [review]

**Pull request merged**